ansible/playbook.yml

```diff
@@ -147,3 +147,11 @@
         state: restarted
       tags:
         - deploy
+    - name: send notification on slack
+      slack:
+        token: "{{ slack.notifications.token }}"
+        channel: "#community_projects"
+        msg: "Successfully deployed new version of website"
```

Comment:
If possible, can you add either a _sha_ or use the _deploy\_version_ variable (with a default to 'master').

Reply:
Good idea. I've changed it and it now looks like this:
`` Successfully deployed website (master 4d3415eca5355f48d25cd67fc786f41bb41921ea) ``